src/main/java/org/homeschoolpebt/app/cli/TransmitterCommands.java

```diff
@@ -98,19 +98,21 @@ private void zipFiles(Map<String, Submission> appIdToSubmission, String zipFileN
 
       appIdToSubmission.forEach((appNumber, submission) -> {
 
-        String fileName = pdfService.generatePdfName(submission);
         Transmission transmission = transmissionRepository.getTransmissionBySubmission(submission);
         String subfolder = transmission.getConfirmationNumber() + "_" + submission.getInputData().get("lastName") + "/";
```

Comment:
For laterdocs we'll want to use the client-provided application number rather than the one in the database row. If we're putting this all in the same ZIP, which seems fine, it might be nice to also mark which of these folders are laterdocs? So: maybe separate the logic for the folder name into pebt vs laterdoc transmissions and it would be something like:

* **pebt** (current): `101111_Dooner/`
* **laterdoc**: `LaterDoc_101111_Dooner_Tom/`

(where the application number for the laterdoc is `submission.getInputData().get("applicationNumber")`)